audio/audio_policy_volumes.xml

```diff
@@ -1,12 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This commit basically imports the stock audio_polict_volumes and default_volume_tables files. Isn't it better then, to just import them from LineageOS directly and remove them from the device tree? Here are the audio configs in LineageOS: https://github.com/LineageOS/android_frameworks_av/tree/lineage-18.0/services/audiopolicy/config and you just add the files in these lines: https://github.com/GPUCode/android_device_nokia_msm8998-common/blob/c7f010f3b8a153c209526fc3ed7e00686b4b7267/msm8998.mk#L141

Reply:
you got a point there. I've just tried all xml configs from LOS 18 and the audio volume is really nice now, but headphones don't work with them and the in-call audio goes to speaker only. Need to investigate.

Reply:
You should try only those 2 files only. Based on the symptoms you described, I suspect you replaced the audio_policy_configuration file too. That file is device specific (contains manufacturer options) and cannot be imported from LineageOS.

Reply:
I did, I replaced all of them from https://github.com/LineageOS/android_frameworks_av/tree/lineage-18.0/services/audiopolicy/config to check it out. I'll do some comparing and see.